ContractExamples/scripts/alert-deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Deploy the alert contract
+#
+# Jure Kukovec, 2024
+#
+# @license
+# [Apache-2.0](https://github.com/freespek/solarkraft/blob/main/LICENSE)
+
+
+set -e
+
+dir=$(cd `dirname $0`; pwd -P)
+
+cd ${dir}/..
+
+NET=testnet
+(soroban network ls | grep -q $NET) || (echo "add testnet via soroban network"; exit 1)
+
+ACCOUNT=alice
+soroban keys address $ACCOUNT || (echo "add the account $ALICE via soroban keys generate"; exit 1)
```

Comment:
Where does `$ALICE` come from?

Reply:
It's like in the [setter-populate.sh](https://github.com/freespek/solarkraft/blob/main/ContractExamples/scripts/setter-populate.sh). The user (or the testing environment) has to create this account.

Reply:
What I meant is that there's no variable `$ALICE`. Only `$ACCOUNT`